src/backend/udev.rs

```diff
@@ -48,11 +49,13 @@ pub struct UdevBackend<
     S: Session + 'static,
     T: UdevHandler<H> + 'static,
 > {
-    devices: Rc<RefCell<HashMap<dev_t, FdEventSource<(DrmDevice<SessionFdDrmDevice>, H)>>>>,
+    _handler: ::std::marker::PhantomData<H>,
```

Comment:
Is there a good reason to keep this type parameter? Otherwise lets just drop it, if it is not required anymore.

Reply:
The reason to keep it is because it appears in the `T: UdevHandler<H>` constraint.

It may be possible to remove it, but I expect it'd require a significant refactor.

Reply:
Well than lets skip that for now.